recipes/parselglossy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "parselglossy" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "25b53a9100067d61e3c8db62aa7ba4eaad4f861a8be82798ec8c66bde8ec650a"
+
+build:
+  number: 0
```

Comment:
This package seems like it can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python